File: week6 - Python/pset6/mario/less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    print_pyramid(height)


# get height from user
def get_height():
    while True:
        height = get_int("Height: ")
        if height > 0 and height <= 8:
            break
    return height


# print pyramid
def print_pyramid(height):
    i = 0
    for el in range(height):
        i += 1
        print(" " * (height - i) + ("#" * i))


if __name__ == "__main__":
    main()

File: week1 - C/pset1/marioless/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
// prompt user for height
    do
    {
        n = get_int("Height: ");

    }
    while (n >= 9 || n <= 0);
// prints left-hand side of pyramid of
    for (int i = 0; i < n; i++)
    {
        for (int x = n - 2; x >= i; x--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
